lib/Installation/Partitioner/LibstorageNG/v4_3/FormatMountOptionsPage.pm

```diff
@@ -136,4 +139,34 @@ sub press_next {
     return $self->{btn_next}->click();
 }
 
+sub edit_fstab_options {
+    my ($self, $mount_by, $default) = @_;
```

Comment:
As we have multiple parameters, I would suggest using hash already. Common practice is to use hash once, there is more than 1 variable. `$self` is special case, of course. Would be nice also if you document what `$default` denotes,

Reply:
Done